### PR TITLE
Did 3 assembly, 21 mods for new 4.0 image content

### PR DESCRIPTION
**Pull request by chrisnegus**
The three assemblies I created were Creating Images (images-create.adoc), Managing image streams (image-streams-managing), and Understand containers, images, and image streams (images-understand).
(Also: Removed unneeded PLACEHOLDER file.)

### CODE REVIEW

**Comment by chrisnegus** [comment]
@openshift/team-documentation I need to have this content peer reviewed.  I didn't do a lot of content change from the 3.11 Image-relate docs. I plan to think that through later. But for now, everything is in the three assemblies and the 21 modules.

**Comment by bmcelvee** [comment]
@chrisnegus, I'll review this morning :) 

Thanks!

**Comment by chrisnegus** [comment]
Thanks Brandi!

----- Original Message -----

File: modules/image-stream-configure.adoc [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+= Configuring image streams
+
+ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
+An image stream object file contains the following elements.
+endif::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
+
+ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
+[NOTE]
+====
+See the xref:../../dev_guide/managing_images.adoc#dev-guide-managing-images[Developer Guide] for details
```
Comment (bmcelvee) [inline modules/image-stream-configure.adoc]: I would recommend removing all xrefs (content may not be available + we're not supposed to use them in the modular docs).

We can note the related content in the additional resources section or if appropriate, include the relevant module.

**Comment by bmcelvee** [comment]
@chrisnegus it's looking pretty good! Just a few additional notes:

The larger modules look like they can be broken down further.
Be sure to call out procedures. 
And add the assembly files and names to the topic map (so they'll build).

**Comment by vikram-redhat** [comment]
@chrisnegus - would you have time this week to address the peer review comments?

**Comment by vikram-redhat** [comment]
@chrisnegus - bump. Can you fix the issues this week so we can merge?

**Pull request merged**